### ISSUE TITLE
Metaboxes not appearing on Gutenberg-enabled Pages/Posts

### ISSUE THREAD

**Issue by timelytree** [htmlburger/carbon-fields#961]
## Version

  - Carbon Fields: 3.2.1
  - WordPress: 5.6
  - PHP: 7.3.5

## Expected Behavior
Pages that use the Gutenberg editor should render any Carbon Fields below the editor itself.

## Actual Behavior
1. _All_ Containers are displayed below the Gutenberg editor - even ones that are tied to other page templates. Screenshot below is of the Blog Single page, but _all_ containers from other pages are appearing:

<img width="836" alt="Screen Shot 2020-12-29 at 12 54 39 PM" src="https://user-images.githubusercontent.com/11708190/103303872-18ed5200-49d5-11eb-8c62-7b0e0fd843d3.png">

The containers are all added like this:

```php
Container::make('post_meta', __('Blog Index Page Options'))
  ->where('post_template', '=', 'page-blog-index.php')
  ->add_fields(array());
```

Which means that this container should not be appearing on the Blog Single page.

2. The containers in the screenshot above are empty, the "toggle" seems to work (I can see the `closed` CSS class being added/removed in the DOM) but the containers themselves are empty.

3. There are potentially relevant errors in my console:
<img width="637" alt="Screen Shot 2020-12-29 at 12 57 15 PM" src="https://user-images.githubusercontent.com/11708190/103303985-71bcea80-49d5-11eb-92ac-b66124f797ff.png">
<img width="640" alt="Screen Shot 2020-12-29 at 12 57 03 PM" src="https://user-images.githubusercontent.com/11708190/103303989-7386ae00-49d5-11eb-9380-c188089006da.png">
<img width="637" alt="Screen Shot 2020-12-29 at 12 57 26 PM" src="https://user-images.githubusercontent.com/11708190/103303988-72ee1780-49d5-11eb-8626-3b93350c3e96.png">

## Comments
This is only an issue on Gutenberg-enabled pages. On pages that use the Classic editor _or_ ones where both editors are completely disabled, Carbon Fields containers/metaboxes appear and work well and with no console errors.

**Comment by kennydude** [comment]
This issue seems similar to #870 :/

**Comment by timelytree** [comment]
@kennydude it really does look similar to #870 *tear

In fact, about 1.5 months ago I was attempting to make changes to the React app (to output more specific classes on the fields and containers so I can target them via an admin-specific CSS sheet) but was running into the same error as in #870 _during the build process_. I downgraded Node as per some of the suggestions in #870 - to no avail.

**Comment by kennydude** [comment]
I did manage to get it building in the end using node v10 and pinning a version of `refract-callbag` which is beyond my very limited React capabilities -- https://github.com/InvisDragon/carbon-fields/commit/b1d4a892e827fcabe77e6ebf262fad07ef078da2

**Comment by jorostoyanov** [comment]
This is now fixed in the latest v3.4.0-beta.1 pre-release - https://github.com/htmlburger/carbon-fields/releases/tag/v3.4.0-beta.1

Please test it out and re-open the issue if you happen to have any issues.
